### ISSUE TITLE
Not working at all

### ISSUE THREAD

**Issue by Walt-Ponpon** [spinagon/ComfyUI-seamless-tiling#8]
Hi,
I installed ComfyUI on a macbook pro M1 max, 64 GB ram.
I tried your nodes with SD3 (sd3_medium_incl_clips.safetensors or sd3_medium_incl_clips_t5xxlfp8.safetensors)
I connected them as displayed in your screenshot and tried to generate some flowers patterns, I got patterns but they never tiled seamlessly.
Positive prompt: "hawaian pattern, watercolor hand painted, seamless tile"
Negative prompt: "text, watermark, photo realistic, visible seams, not repeating"

Is it expected or any idea what can be wrong?

**Comment by spinagon** [comment]
Yeah, it won't work on SC3 or Flux, they have a different architecture

**Comment by alexisrolland** [comment]
It works perfectly fine for me with a Flux finetuned model. I am using this model: https://civitai.com/models/1056935/rayflux

I generate a texture image like this:

> seamless, tileable texture of old, broken stone with moss. The texture should feature irregular stone surfaces with cracks and crevices, and patches of moss growing in and on the stones. Ensure high-resolution details with natural variations in color and texture. Optimize for PBR workflows with neutral lighting. Include subtle imperfections like dirt and grime for realism. The mood should be dark and gritty, suitable for fantasy or historical video game environments.

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/7b357832-b4e2-4345-a8af-76049f2cfbf7" />

Here is the same image offset by 50%:

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/b8c10009-e72e-4b05-89cb-40caaa738a27" />

Here is my workflow:

<img width="2360" height="1396" alt="Image" src="https://github.com/user-attachments/assets/2bad88c7-dd49-4302-9f2c-3faa42b98b80" />
